### ISSUE TITLE
Fix epydoc pydoctor errors in twisted.python

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8554]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#8554|
|Type|enhancement|
|Created|2016-07-03 16:13:55Z|
|Branch|https://github.com/twisted/twisted/tree/8554-pydoctor-t-python|

<details><summary>Searchable metadata</summary>

```
trac-id__8554 8554
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__None None
branch__8554_pydoctor_t_python 8554-pydoctor-t-python
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467562435755710 1467562435755710
changetime__1467726039048760 1467726039048760
version__None None
owner__adiroiban adiroiban

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban removed owner|
|-|-|

1. inspect.ArgSpec is a named tuple and has no API documentation 

2. There were a couple of cross referenced refering to an argument name of a method. I have converted them to I{} as the argument names can not be referenced.

3. There is no doc for socket.errorTab

4. I had no idea why I got this error `twisted.python._textattributes._ColorAttribute:171 invalid ref to _ColorAttribute.attrs` so I made it an I{} markup since the reference is just there

--------

here is my local output 

```
$ ./bin/admin/build-apidocs . apidocs/ | grep twisted.python

twisted.python.win32._ErrorFormatter:106 invalid ref to win32api.FormatMessage
twisted.python.win32._ErrorFormatter.formatMessage:0 invalid ref to win32api.FormatMessage
twisted.python.components._addHook:107 invalid ref to zope.interface.adapter.AdapterRegistry
twisted.python.components._addHook:107 invalid ref to zope.interface.adapter.AdapterRegistry

```


---------

win32api.FormatMessage was left as we can not yet link to pywin32 API and this should be done in a separate ticket. Maybe we can do it similar to zope with custom URL [[#8515](https://github.com/twisted/twisted/issues/8515)](https://github.com/twisted/twisted/issues/8515)


zope.interface.adapter.AdapterRegistry is also fixed in #8515

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc set owner to @adiroiban|
|-|-|

OK

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset 1dec4463f9340712bb338488d77eb4ed0130a24a
```
#!CommitTicketReference repository="" revision="1dec4463f9340712bb338488d77eb4ed0130a24a"
Merge 8554-pydoctor-t-python: Fix pydoctor errors in twisted.python.

Author: adiroiban
Reviewer: rodrigc
Fixes: #8554
```
